src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processor/CourtCaseEventsProcessor.kt

```diff
@@ -83,7 +83,20 @@ class CourtCaseEventsProcessor(
         mapOf("PNC" to person.otherIdentifiers?.pncIdentifier?.pncId, "CRO" to person.otherIdentifiers?.cro),
       )
 
-      courtCaseEventsService.processPersonFromCourtCaseEvent(person)
+      retry { courtCaseEventsService.processPersonFromCourtCaseEvent(person) }
+    }
+  }
+  fun retry(
+    block: () -> Unit,
+  ) {
+    repeat(2) {
```

Comment:
This is 0 based, should probably be a constant  named `THREE_TIMES` or something similar